buildSrc/src/main/kotlin/scripts/detekt.gradle.kts

```diff
@@ -0,0 +1,61 @@
+package scripts
+
+import com.wire.kalium.plugins.libs
+import getLocalProperty
+import io.gitlab.arturbosch.detekt.Detekt
+
+plugins {
+    id("io.gitlab.arturbosch.detekt")
+}
+
+dependencies {
+    val detektVersion = libs.findVersion("detekt").get()
+    detekt("io.gitlab.arturbosch.detekt:detekt-cli:$detektVersion")
+    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:$detektVersion")
+    detektPlugins("com.wire:detekt-rules:1.0.0-SNAPSHOT") {
+        isChanging = true // enforce snapshot version check for updates
+    }
+}
+
+configurations {
+    all {
+        resolutionStrategy {
+            eachDependency {
+                if (requested.group == "com.wire" && requested.name == "detekt-rules") {
+                    // enforce snapshot version check for updates everytime instead of 24 hrs.
+                    cacheChangingModulesFor(0, "SECONDS")
```

Comment:
Doesn't that prevent people from working offline?

Reply:
Yup, that's correct. I removed it, so we get the default cache behavior of 24 hrs. If we need to test faster because of changes, we can use `mavenLocal` as suggested and also added

Reply:
I will add in the rules repo now the complementary part:
- publish plugin to publish to local
- automation to tag versions

Reply:
I added to https://github.com/wireapp/wire-detekt-rules : 
- the publish plugin, so we can test locally
- the gh action to update and tag an updated artifact

:D